client/actions/excludes.js

```diff
@@ -4,24 +4,24 @@ import * as constants from '../constants';
 /*
  * Load the rules.
  */
-export function fetchAllRules() {
+export function fetchExcludes() {
```

Comment:
why did we rename the rules file when adding multiple excludes?

Reply:
Originally, there weren't any other excludes beside rules. Then resourceServers was added and we just copied excludeRules component. Now we have rules, resourceServers, clients, connections and databases and all are using the same code. So i removed `resourceServers` excludes component and changed `rules` to general `excludes`.